ch02/README.md

```diff
@@ -437,7 +437,7 @@ i = ic;     // legal.
 p1 = p3;    // illegal. p3 is a pointer to const int.
 p1 = &ic;   // illegal. ic is a const int.
 p3 = &ic;   // legal.
-p2 = p1;    // legal.
```

Comment:
p2 hasn't been initialized, which is an error in `Exercise 2.28` .when we do `Exercise 2.29`,we have to assume `p2` had been initialized correctly, right? After all, it's not allowed to assign a value to a const pointer.